pkg/apis/extensions/v1alpha1/rpaasinstance_types.go

```diff
@@ -28,6 +29,12 @@ type RpaasInstanceSpec struct {
 	// nginx config.
 	Locations []Location `json:"locations,omitempty"`
 
+	// Certificates are a set of attributes that relate the certificate's
+	// location in the cluster (Secret resource name) and its destination into
+	// Pods.
+	// +optional
+	Certificates map[CertificateName]nginxv1alpha1.TLSSecret `json:"certificates,omitempty"`
```

Comment:
No, it does not need. I did it because it could make easier our work on future to accept many certificates in a single instance. There are some use cases for that, such as multiple certificates by server block (serve RSA or ECDSA certificate based on TLS negotiation), or different ones based on server name, etc.